scu-es/src/fiskaltrust.Middleware.SCU.ES.TicketBAI/Models/TicketBaiRequest.cs

```diff
@@ -0,0 +1,1188 @@
+
+        [RegularExpression("(\\+|-)?\\d{1,12}(\\.\\d{0,2})?")]
+        [Required]
+        [XmlElement("CuotaRectificada")]
+        public string CuotaRectificada { get; set; }
+
+        [RegularExpression("(\\+|-)?\\d{1,12}(\\.\\d{0,2})?")]
+        [XmlElement("CuotaRecargoRectificada")]
+        public string CuotaRecargoRectificada { get; set; }
+    }
+
+    [Serializable]
+    [XmlType("FacturasRectificadasSustituidasType", Namespace = "urn:ticketbai:emision")]
+    public class FacturasRectificadasSustituidasType
+    {
+        [Required]
+        [XmlElement("IDFacturaRectificadaSustituida")]
+        public List<IDFacturaRectificadaSustituidaType> IDFacturaRectificadaSustituida { get; set; }
+    }
+
+    [Serializable]
+    [XmlType("IDFacturaRectificadaSustituidaType", Namespace = "urn:ticketbai:emision")]
+    public class IDFacturaRectificadaSustituidaType
+    {
+        [MaxLength(20)]
+        [XmlElement("SerieFactura")]
+        public string SerieFactura { get; set; }
+
+        [MaxLength(20)]
+        [Required]
+        [XmlElement("NumFactura")]
+        public string NumFactura { get; set; }
+
+        [MinLength(10)]
+        [MaxLength(10)]
+        [RegularExpression("\\d{2,2}-\\d{2,2}-\\d{4,4}")]
+        [Required]
+        [XmlElement("FechaExpedicionFactura")]
+        public string FechaExpedicionFactura { get; set; }
+    }
+
+
+    [Serializable]
+    [XmlType("DatosFacturaType", Namespace = "urn:ticketbai:emision")]
+    public class DatosFacturaType
+    {
+        [MinLength(10)]
+        [MaxLength(10)]
+        [RegularExpression("\\d{2,2}-\\d{2,2}-\\d{4,4}")]
+        [XmlElement("FechaOperacion")]
+        public string FechaOperacion { get; set; }
+
+        [MaxLength(250)]
+        [Required]
+        [XmlElement("DescripcionFactura")]
+        public string DescripcionFactura { get; set; }
+
+        [XmlArray("DetallesFactura")]
+        [XmlArrayItem("IDDetalleFactura")]
+        public List<IDDetalleFacturaType> DetallesFactura { get; set; }
+
+        [RegularExpression("(\\+|-)?\\d{1,12}(\\.\\d{0,2})?")]
+        [Required]
+        [XmlElement("ImporteTotalFactura")]
+        public string ImporteTotalFactura { get; set; }
+
+        [RegularExpression("(\\+|-)?\\d{1,12}(\\.\\d{0,2})?")]
+        [XmlElement("RetencionSoportada")]
+        public string RetencionSoportada { get; set; }
+
+        [RegularExpression("(\\+|-)?\\d{1,12}(\\.\\d{0,2})?")]
+        [XmlElement("BaseImponibleACoste")]
+        public string BaseImponibleACoste { get; set; }
+
+        [Required]
+        [XmlArray("Claves")]
+        [XmlArrayItem("IDClave")]
+        public List<IDClaveType> Claves { get; set; }
+    }
+
+    [Serializable]
+    [XmlType("DetallesFacturaType", Namespace = "urn:ticketbai:emision")]
+    public class DetallesFacturaType
+    {
+        [Required]
+        [XmlElement("IDDetalleFactura")]
+        public List<IDDetalleFacturaType> IDDetalleFactura { get; set; }
+    }
```

Comment:
It looks like it may be duplicated to lines 656-658 as the List<> is declared there too.